01week/datatypes.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+// 1. Write a JavaScript program to display the current day and time.
+function displayDateAndTime() {
+  return Date();
+}
+
+console.log('\n\ndisplayDateAndTime');
+console.log('------------------');
+console.log('The current date and time is: ', displayDateAndTime());
+
+
+
+// 2. Write a JavaScript program to convert a number to a string.
+function convertNumberToString(num) {
+  // ensure num is really a number, to handle strings, nulls, etc
```

Comment:
what would you do if num isn't a number?

Reply:
The requirement reads "convert a number to a string" and does not define how exceptional cases should be handled, so I chose to handle arguments that are "not a number" by returning the string 'NaN'.  This is accomplished by first casting the incoming value using Number().  Invalid values, such as null, undefined, objects, etc, will produce NaN, which then becomes 'NaN' when toString() is called.